vertical-pod-autoscaler/pkg/admission-controller/main.go

```diff
@@ -48,12 +48,13 @@ var (
 		tlsPrivateKey: flag.String("tls-private-key", "/etc/tls-certs/serverKey.pem", "Path to server certificate key PEM file."),
 	}
 
-	port           = flag.Int("port", 8000, "The port to listen on.")
-	address        = flag.String("address", ":8944", "The address to expose Prometheus metrics.")
-	namespace      = os.Getenv("NAMESPACE")
-	webhookAddress = flag.String("webhook-address", "", "Address under which webhook is registered. Used when registerByURL is set to true.")
-	webhookPort    = flag.String("webhook-port", "", "Server Port for Webhook")
-	registerByURL  = flag.Bool("register-by-url", false, "If set to true, admission webhook will be registered by URL (webhookAddress:webhookPort) instead of by service name")
+	port                = flag.Int("port", 8000, "The port to listen on.")
+	address             = flag.String("address", ":8944", "The address to expose Prometheus metrics.")
+	namespace           = os.Getenv("NAMESPACE")
+	webhookAddress      = flag.String("webhook-address", "", "Address under which webhook is registered. Used when registerByURL is set to true.")
+	webhookPort         = flag.String("webhook-port", "", "Server Port for Webhook")
+	registerByURL       = flag.Bool("register-by-url", false, "If set to true, admission webhook will be registered by URL (webhookAddress:webhookPort) instead of by service name")
+	allowToAdjustLimits = flag.Bool("allow-to-adjust-limits", false, "If set to true, admission webhook will set limits per container too if needed")
```

Comment:
Actually this flag does not seem necessary any more. We are already adjusting limits even when this flag is set to false (see your previous PRs) so this is not accurate. I think we can remove it and also get rid of NoopLimitsCalculator - if we already manipulate limits we might as well always observe LimitRange.

Reply:
Done.